docs/source/tutorials/include/scripts/run_md_basic.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env runaiida
+import io
+
+from aiida.engine import run
+from aiida.orm import Dict, StructureData, load_code
+from ase.build import bulk
+import requests
+
+from aiida_lammps.data.potential import LammpsPotentialData
+
+# Load the code configured for ``lmp``. Make sure to replace
+# this string with the label used in the code setup.
+code = load_code("lammps@localhost")
+builder = code.get_builder()
+
+structure = StructureData(ase=bulk("Fe", "bcc", 2.87, cubic=True))
+builder.structure = structure
+
+# Download the potential from the repository and store it as a BytesIO object
+_stream = io.BytesIO(
+    requests.get(
+        "https://openkim.org/files/MO_546673549085_000/Fe_2.eam.fs", timeout=20
+    ).text.encode("ascii")
```

Comment:
is `ascii` enconding intentional? Shouldn't we just use utf-8, as is the standard across all of AiiDA?

Reply:
I just used ascii as I was not sure, but I think that utf8 should be good.